t/01_version_utils.t

```diff
@@ -13,6 +13,25 @@ use lib ("$FindBin::Bin/lib", "$FindBin::Bin/../lib");
 
 use testapi qw(check_var get_var set_var);
 
+subtest 'check_version' => sub {
+    # compare versions if possible
+    ok version_utils::check_version($_, '15.5'), "check $_, 15.5" for qw[ >15.0 <15.10 ];
```

Comment:
What's about adding test for our leap naming (42.3<15.0)?

Reply:
This check already exists, look a bit lower at [`subtest 'is_leap'`](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/01_version_utils.t#L36)

I moved leap versioning hacks (13 < 42 < 15) to `is_leap` sub, because it's leap specific.

Reply:
Ah, sorry, missed that.